src/lib/daemon_rpcs/daemon_rpcs.ml

```diff
@@ -512,6 +512,17 @@ module Snark_job_list = struct
     Rpc.Rpc.create ~name:"Snark_job_list" ~version:0 ~bin_query ~bin_response
 end
 
+module Snark_pool_list = struct
```

Comment:
We are not trying to create RPCs anymore. Can you write `Snark_pool_list` as a graphql service?

Reply:
Why don't we want to create RPCs anymore? Also, this is mainly for next week's snark challenge.